### DISCUSSION TITLE
Cinnamon: Battery Warning Notifications

### DISCUSSION THREAD

**Discussion by trymeouteh** [linuxmint/discussions#568] [Ideas]
To have Cinnamon settings that will push a notification when the battery level reaches a certain percentages that the user can choose (50%, 40%, 30%, 20%, etc)

**Comment by rcalixte** [comment]
There is already an applet for this named [Battery Applet with Monitoring and Shutdown (BAMS)](https://cinnamon-spices.linuxmint.com/applets/view/255). It also has an option to alert at a recharge level as well. You can adjust the discharge threshold for alerting starting at 40%. You can install it using the built-in settings or you can download it from the site.
